unified_planning/engines/plan_validator.py

```diff
@@ -73,6 +73,7 @@ def supported_kind() -> ProblemKind:
         supported_kind.set_conditions_kind("EQUALITY")
         supported_kind.set_conditions_kind("EXISTENTIAL_CONDITIONS")
         supported_kind.set_conditions_kind("UNIVERSAL_CONDITIONS")
+        supported_kind.set_constraints_kind("TRAJECTORY_CONSTRAINTS")
```

Comment:
Are the trajectory constraints really supported by the plan validator?

Reply:
hi @alvalentini, I had already fixed it according to the comments made. 
Regarding plan_validator, I forgot to delete the line after trying it, and no, not support native traj_constr